lib/turbo_tests/runner.rb

```diff
@@ -157,13 +160,16 @@ def start_subprocess(env, extra_args, tests, process_id, record_runtime:)
             []
           end
 
+        spec_opts = ["-O", options_file] if options_file
```

Comment:
Let's use `parallel-tests` directly.

```suggestion
        spec_opts = ParallelTests::RSpec::Runner.send(:spec_opts)
```